unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ViewController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ViewController : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+}
```

Comment:
Is this class used somewhere or can we delete the file? otherwise at least we can remove those empty mehods